workshops/models.py

```diff
@@ -1085,10 +1085,15 @@ def __str__(self):
 class Role(models.Model):
     '''Enumerate roles in workshops.'''
 
-    name       = models.CharField(max_length=STR_MED)
+    name = models.CharField(max_length=STR_MED)
+    verbose_name = models.CharField(max_length=STR_LONG,
+                                    null=False, blank=True, default='')
+    show_in_training_request_form = models.BooleanField(
+        default=False,
+        verbose_name='include in training request form?')
```

Comment:
It's a way to control whether a role is an option under "Previous involvement with Software Carpentry or Data Carpentry" question. There are two roles that are not included in the form: `debriefed` and `tutor`, both roles will be deleted.